### ISSUE TITLE
Question about compiling sample-mail

### ISSUE THREAD

**Issue by TheOska** [sockeqwe/mosby#243]
I cannot run mosby with some error message 
![screen shot 2017-05-05 at 12 06 08 pm](https://cloud.githubusercontent.com/assets/11329673/25733477/02b87272-318c-11e7-856b-39c61c965e94.png)

I tried "clear project" and "rebuild project" but fail to run, Is my setting problem? Thanks.

**Comment by chennyshan** [comment]
+1

**Comment by sockeqwe** [comment]
Hm, not sure why it doesn't compile. On my machine it compiles just fine and also on Travis CI.
could you try to build it from command line:
```
./gradlew clean build
```
It should build successfully. Then you can try to run the app from android studio (green play button).

Let me know if this helps.

**Comment by chennyshan** [comment]
I tried the previous command line, but got the same error as that from android studio. FYI
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sample-mail:compileDebugJavaWithJavac'.
> java.lang.NoSuchFieldError: MAP

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by sockeqwe** [comment]
hm, compiles on my machine from command line.

<img width="536" alt="bildschirmfoto 2017-05-17 um 10 22 41" src="https://cloud.githubusercontent.com/assets/583598/26144882/d8ec95b4-3aea-11e7-98d7-730ae0a02b58.png">

Have you changed some code or gradle plugin version?

**Comment by sockeqwe** [comment]
Ok, I'm now finally able to reproduce this issue on circle ci

https://circleci.com/gh/sockeqwe/mosby/1

I hope to find some time to fix this issue in August

**Comment by BrianOdisho** [comment]
@sockeqwe any general idea on what could be causing this issue?

**Comment by sockeqwe** [comment]
not really, but I will replace the annotation processor that causes this issue
